.github/workflows/code-checks.yml

```diff
@@ -85,5 +85,24 @@ jobs:
 
       - run: pnpm build
 
-      - name: Run unit tests
+      - name: Run unit tests with code coverage
```

Comment:
I wonder if we still need `test-unit` as here I think we're running the tests twice?

Reply:
It's a good question, likely we don't, though I split them for the sake of the initial testing (at the very least) for a bit of sanity. I should probably remove the `test-coverage` command  and just stick with `unit` for the ci